sample_apps/simple_app_under_oe/instructions.md

```diff
@@ -157,15 +157,22 @@ $CERTIFIER_PROTOTYPE/utilities/package_claims.exe     \
 $CERTIFIER_PROTOTYPE/utilities/print_packaged_claims.exe --input=policy.bin
 ```
 
-## Step 8: Use Policy Generator
+## Step 8: Use Automated Policy Generator
 
 ### a. Edit policy file
 
 Open `simple_app_under_oe/oe_policy.json` and replace trusted measurements
 in the "measurements" property with the expected measurement from
 `make dump_mrenclave`.
 
-### b. Run Policy Generator
+### b. Build the Policy Generator
+
+```shell
+cd $CERTIFIER_PROTOTYPE/utilities
+LOCAL_LIB=/usr/local/lib64 make -f policy_generator.mak
```

Comment:
This should work on the test environment we have. However, dependencies needs to be built and installed before the generator build. This is described in the note in utilites.

Reply:
HI, @yelvmw - I'm not sure what to make of this comment.

Are you agreeing with this change, and just stating something for the record?

Or, are you suggesting that there needs to be some more / other changes in this change-set to complete the docs for this work?

Unclear ... Can you let me know if you see any further need for code / docs changes?

Reply:
I'm just saying the make command will fail for anyone who did not install the dependencies. And at least one of the libraries needs to be built manually. As long as this is expected, I'm fine with the change.

Reply:
Let me see if I understand this, @yelvmw :

- policy_generator.mak references : `LDFLAGS= -L$(LOCAL_LIB) -lnlohmann_json_schema_validator -lgflags`
- The file utilities/policy_generator_info.txt talks about:
```
  3 You will need the json-schema-validator library which in turn depends on the
  4 JSON for Modern C++ library. Follow the following instructions to have them
  5 built and installed on your system:
  ```
  
I see that on this machine the required JSON library is installed.

And, by this: _And at least one of the libraries needs to be built manually_. -- if you mean `libnlohmann_json_schema_validator.so`, that's already installed on this machine.

Seems like there is, yet, another set of pre-installation setup stuff one needs to go through, and someone has done that on this machine.
 
I think the AI here is as follows:
- Rewrite policy_utilities_info.txt and policy_generator_info.txt as md-files
- There already exists [INSTALL.md in utilities](https://github.com/vmware-research/certifier-framework-for-confidential-computing/blob/main/INSTALL.md)/ . Maybe some of the pre-setup installation stuff needed for JSON libraries could go in there? Not sure ... need to discuss.

So, my assessment is that the changes submitted by themselves are ok, but there is some need to massage the docs into md-files highlighting the pre-setup installation steps needed.

Reply:
Correct. I think your change is OK. Let me update INSTALL.md in utilities. Thanks.